letsencrypt/models/letsencrypt.py

```diff
@@ -75,7 +75,10 @@ def generate_domain_key(self, domain):
 
     @api.model
     def validate_domain(self, domain):
-        local_domains = ['localhost', 'localhost.localdomain']
+        local_domains = [
```

Comment:
Although it is OK and these are just 4 elements, I think it could be good to separate this list in another method. Also if you use a `set()` instead of a list, you get a performance boost for `in` operator. Not a blocker, just a tip you might find useful :wink: